modules/GKB/Config_Species.pm

```diff
@@ -8,193 +8,279 @@ use Exporter();
-                                        'access' => 'http://fungi.ensembl.org/Schizosaccharomyces_pombe/Transcript/ProteinSummary?peptide=###ID###',
-					'ensg_access' => 'http://fungi.ensembl.org/Schizosaccharomyces_pombe/geneview?gene=###ID###&db=core'},
-                            'alt_refdb' => {'dbname' => ['GeneDB'],
-                                        'url' => 'http://www.genedb.org/genedb/pombe',
-                                        'access' => 'http://www.genedb.org/genedb/Search?organism=pombe&name=###ID###'},
-                            'group' => 'Fungi/Plants',
-                            'mart_url' => 'http://fungi.ensembl.org/biomart/martservice',
-                            'mart_virtual_schema' => 'fungi_mart',
-                            'mart_group' => 'spombe_eg_gene'},
-		 'sscr' => {'name' => ['Sus scrofa'],
-			    'refdb' => {'dbname' => ['ENSEMBL', 'Ensembl', 'ENSEMBL_Sus_scrofa_PROTEIN'],
-                                        'url' => 'http://www.ensembl.org/Sus_scrofa/Info/Index/',
-                                        'access' => 'http://www.ensembl.org/Sus_scrofa/Transcript/ProteinSummary?peptide=###ID###',
-                                        'ensg_access' => 'http://www.ensembl.org/Sus_scrofa/geneview?gene=###ID###&db=core'},
-                            'group' => 'Vertebrate',
-                            'mart_group' => 'sscrofa_gene_ensembl',
-                            'compara' => 'core'},
-        'tgut' => {'name' => ['Taeniopygia guttata'],
-                            'refdb' => {'dbname' => ['ENSEMBL', 'Ensembl', 'ENSEMBL_Taeniopygia_guttata_PROTEIN'],
-                                        'url' => 'http://www.ensembl.org/Taeniopygia_guttata/Info/Index/',
-                                        'access' => 'http://www.ensembl.org/Taeniopygia_guttata/Transcript/ProteinSummary?peptide=###ID###',
-					'ensg_access' => 'http://www.ensembl.org/Taeniopygia_guttata/geneview?gene=###ID###&db=core'},
-                            'group' => 'Vertebrate',
-                            'mart_group' => 'tguttata_gene_ensembl',
-			    'compara' => 'core'},
-		 'xtro' => {'name' => ['Xenopus tropicalis'],
-			    'refdb' => {'dbname' => ['ENSEMBL', 'Ensembl', 'ENSEMBL_Xenopus_tropicalis_PROTEIN'],
-                                        'url' => 'http://www.ensembl.org/Xenopus_tropicalis/Info/Index/',
-                                        'access' => 'http://www.ensembl.org/Xenopus_tropicalis/Transcript/ProteinSummary?peptide=###ID###',
-					'ensg_access' => 'http://www.ensembl.org/Xenopus_tropicalis/geneview?gene=###ID###&db=core'},
-                            'group' => 'Vertebrate',
-                            'compara' => 'core',
-                            'mart_group' => 'xtropicalis_gene_ensembl'},
-		 );
+@species = qw(hsap ddis pfal spom scer cele sscr btau cfam mmus rnor ggal xtro drer dmel);
 
+%species_info = (
```

Comment:
Could we just delete these now-unused species?

Reply:
Yeah, I was just commenting them in case we had to include them again but we've got them in history.  I'll remove them.